File: test8_out/test8/HelloWorld/HelloWorld.c

/*
 * ======== string literals ========
 */

static const char test8_HelloWorld__s__fileName[] = "HelloWorld.p";
static const char test8_HelloWorld__s__0[] = "HelloWorld";

/*
 * ======== function definitions ========
 */

void test8_HelloWorld_pollen__run__E() {
    test8_HelloWorld_start__I((test8_HelloWorld_e1__V));
    test8_HelloWorld_start__I((test8_HelloWorld_e2__V));
}

bool test8_HelloWorld_start__I( struct test8_Event*  e ) {
    uint8 code;
    code = test8_Event_Event_getCode__E(e);
    return(  true  );
}

void test8_HelloWorld_handle__E( uint8 z ) {
    uint8 x;
    x = 3;
}

void test8_HelloWorld_targetInit__I() {
}

File: test8_out/test8/HelloWorld/HelloWorld.js

/*
 * ======== MODULE test8.HelloWorld ========
 */

var $$c = function() {
    const HelloWorld = {};
    HelloWorld.HelloWorld = HelloWorld
    HelloWorld.$name = 'test8.HelloWorld';
    HelloWorld.pollen$used = 0;
    HelloWorld.Event = $units['test8.Event'];
    HelloWorld.start = new $$Ref('test8_HelloWorld_start__I');
    HelloWorld.pollen__run = new $$Ref('test8_HelloWorld_pollen__run__E');
    HelloWorld.handle = new $$Ref('test8_HelloWorld_handle__E');
    HelloWorld.targetInit = new $$Ref('test8_HelloWorld_targetInit__I');
    HelloWorld.$$hostInit = function() {
        var $$text = '';
        return $$text;
    }
    HelloWorld.pollen__uses__ = function() {
        $$bind($units['test8.Event'], 'pollen$used', true);
    }
    HelloWorld.$$privateInit = function() {
    }
    return HelloWorld;
}
var $$u = $$c();
$units['test8.HelloWorld'] = $$u;
$units.push($$u);

